components/src/Toast/Toast.story.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { storiesOf } from "@storybook/react";
+import { Button, DangerButton, IconicButton, PrimaryButton } from "../Button";
+import { Flex } from "../Flex";
+import { Toast } from ".";
+
+storiesOf("Toast", module)
+  .add("Toast", () => {
+    const ToastWithTrigger = () => {
+      const [triggered, setTriggered] = useState(false);
+      const triggerToast = () => {
+        setTriggered(!triggered);
+      };
+      const onHideHandler = () => {
+        setTriggered(false);
+      };
+      return (
+        <>
+          <Button onClick={triggerToast}>Save Changes</Button>
+          <Toast triggered={triggered} onHide={onHideHandler}>
+            Saved
+          </Toast>
+        </>
+      );
+    };
+    return <ToastWithTrigger />;
+  })
+  .add("multiple toasts example", () => {
+    const MultipleToastsExample = () => {
+      const [currentToast, setCurrentToast] = useState(undefined);
+      const triggerToast = toastName => {
+        setCurrentToast(toastName);
+      };
+      /* Alternate trigger toast fn adds a delay between toasts
```

Comment:
I'm not sure about whether this delay is needed. I tried but waiting for the orginal toast to fade out makes clicking the buttons feel less responsive.. lmk your thoughts

Reply:
Honestly, I can't tell the difference.